src/lucky/tags/base_tags.cr

```diff
@@ -75,14 +75,22 @@ module Lucky::BaseTags
     end
   end
 
+  private def build_boolean_attrs(options)
+    String.build do |attrs|
+      options.each do |value|
+        attrs << " " << LuckyInflector::Inflector.dasherize(value.to_s)
+      end
+    end
+  end
+
   private def merge_options(html_options, tag_attrs)
     options = {} of String => String
     tag_attrs.each do |key, value|
       options[key.to_s] = value
     end
 
     html_options.each do |key, value|
-      options[key.to_s] = value
+      options[key.to_s] = value.as(String)
```

Comment:
I had to put this here (for now at least) because I'm getting 
> no overload matches 'Hash(String, String)#[]=' with types String, (Array(Symbol) | String)

At some point, the compiler thinks `value` could have `Array(Symbol)`. This is from passing the `attrs: [:required]` in to the `html_options`. Though, that array never actually gets in here